api/validator.js

```diff
@@ -44,7 +45,8 @@ const SCHEMA_OUTPUT = Joi.object()
     .keys({
         jobs: SCHEMA_JOBS,
         workflow: Workflow.workflow,
-        errors: Joi.array().items(Joi.string()).optional()
+        errors: Joi.array().items(Joi.string()).optional(),
+        annotations: Annotations.annotations
```

Comment:
@d2lam @stjohnjohnson updated. It's now in both `SCHEMA_JOB_PERMUTATION` (for job-level) and `SCHEMA_OUTPUT` (for pipeline-level)